packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -104,18 +107,20 @@ export class CheckpointService implements TreeDataProvider<BaseNode> {
         this.orgInfo = await new ForceOrgDisplay().getOrgInfo(this.sfdxProject);
       } catch (error) {
         const result = JSON.parse(error) as OrgInfoError;
-        vscode.window.showErrorMessage(
-          nls.localize('unable_to_retrieve_org_info') + ': ' + result.message
-        );
+        const errorMessage = `${nls.localize(
+          'unable_to_retrieve_org_info'
+        )} : ${result.message}`;
+        writeToDebuggerOutputWindow(errorMessage);
+        vscode.window.showErrorMessage(errorMessage);
```

Comment:
Could we combine `writeToDebuggerOutputWindow(errorMessage);` and `vscode.window.showErrorMessage(errorMessage);` to a single method? I see these two paired up multiple times.

Reply:
They aren't paired up everywhere but you're right, I'll add a boolean flag to writeToDebuggerOutputWindow that'll control whether or not to display an error message.